### PR TITLE
Bug 1135233 - Made sure '#statusbar-tray' icons are accessible when utility tray is open

### PR DESCRIPTION
**Pull request by funktr0n**

### CODE REVIEW

File: apps/system/style/zindex.css [hunk 1]
```diff
@@ -105,6 +105,11 @@
   z-index: 16385;
 }
```
Comment (yzen) [inline apps/system/style/zindex.css]: I would just add a comment why this is needed.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bbe27e2c310aaa3081e885fa2c7a4349c3424ea7

File: apps/system/style/zindex.css [hunk 2]
```diff
@@ -105,6 +105,16 @@
...
 
+/* We need to have the statusbar lying on top of the utility tray
+   when it's open so that its icons are are accessible to touch events
```
Comment (yzen) [inline apps/system/style/zindex.css]: nit: typo - 'are are'

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=669706a2724294d9e9f6c9948175d15cc95771f8

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79559dc1ae54343127e1ca55d7f030f1429bcac0

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=150c809220d1c8495b9ad9cb41eaed0ccdb25c16

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=117f2eac36339b7b064abe2bea7bfdb9869eda94

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#6RQ6zYSFTyWlIu2_WsyWUg

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a771e543d7121bccfa9c3f929e123b84c1740d81

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=684b48c4c51f7a7474116fbdcbca6623b4a0e718

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d5062c82847bc50c6e7cf8682dcac0db13441968

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=647f38856a46a026e5e4d8c900a242e372f33925

**Pull request merged**